client/components/foldable-card/index.jsx

```diff
@@ -85,10 +86,12 @@ var FoldableCard = React.createClass( {
 			);
 		}
 		if ( this.props.children ) {
-			let iconSize = 24;
+			const iconSize = 24;
+			const screenReaderDefaultText = ( typeof this.translate === 'function' ) ? this.translate( 'More' ) : '';
```

Comment:
`this.translate` is not defined when calypso components like FoldableCard are used in a context (e.g. another project) where https://github.com/Automattic/wp-calypso/tree/master/client/boot is not wanted